sapphire/tests/transformations/test_celestial.py

```diff
@@ -159,5 +161,323 @@ def test_against_jpl_horizons(self):
         self.assertAlmostEqual(dec, dec_astropy, 2)
 
 
+def test_transformations():
+    """
+    This function numerically evaluates a number of preprogrammed
+    values and the difference between our new and old transformations
+    New values may be added to testcases
+    :return: None
+
+    Ethan van Woerkom is responsible for the benchmarking functions; refer to him for when something is unclear
+    """
+    # This tuple list contains all the used testcases for the test.
+    # The meanings of the tuple entries are:
+    # 0: latitude (deg) 1: longitude (deg) 2: utc_timestamp 3: J2000 RA (rad)
+    # 4: J2000 DEC (rad) 5: horizontal az. (rad) 6: horizontal alt. (rad)
+    # 7: source of test name
+    testcases = [(48.86, 2.34, 1497003786, 0.8381070981, -0.5059006522, 3.26454625, 0.207694181, "stell. paris, alpha-for"),
+                 (48.86, 2.34, 1497256975, 1.549728749, 0.1292784768, 1.969511946, 0.4898557422, "stell. paris, betelgeuse"),
+                 (31.9633, -111.6, 2271646799, 0.05388704, 0.2650006125, 4.623697166, 0.66167856, "hor2eq IDL")]
+    ARC_RAD = 2*np.pi / 360 / 3600 # one arcsecond in rad
+    for i in testcases:
+        print 'Transformation test using:'
+        print 'lat, long:', i[0], i[1], 'utc_time:', i[2], 'RA, DEC:', i[3], i[4]
+        print 'az, alt:', i[5], i[6], 'from source: ', i[7]
+
+        t = celestial.equatorial_to_horizontal_astropy(i[0], i[1], i[2], [(i[3], i[4])])
+        print "equatorial_to_horizontal_astropy gave: ", t[0]
+        print "Should have been:                      ", (i[5],i[6])
+        print "Difference (dAZ, dALT) (arcsec):       ", ((t[0][0]-i[5])/ARC_RAD,(t[0][1]-i[6])/ARC_RAD)
+
+        t = celestial.horizontal_to_equatorial_astropy(i[0], i[1], i[2], [(i[5], i[6])])
+        print "horizontal_to_equatorial_astropy gave: ", t[0]
+        print "Should have been:                      ", (i[3],i[4])
+        print "Difference (dAZ, dALT) (arcsec):       ", ((t[0][0]-i[3])/ARC_RAD,(t[0][1]-i[4])/ARC_RAD)
+
+        # equatorial to horizontal is actually a zenithazimuth to horizontal
+        t = celestial.equatorial_to_horizontal(i[0], i[1], clock.utc_to_gps(i[2]), i[3], i[4])
+        print "equatorial_to_horizontal gave:         ", (t[1], t[0])
+        a,b = celestial.horizontal_to_zenithazimuth(i[6], i[5])
+        print "Should have been:                      ", (b,a)
+        print "Difference (dAZ, dALT) (arcsec):       ", ((t[1] - b) / ARC_RAD, (t[0] - a) / ARC_RAD)
+
+        t = celestial.horizontal_to_equatorial(i[0], clock.utc_to_lst(datetime.datetime.utcfromtimestamp(i[2]), i[1]), i[6], i[5])
+        print "horizontal_to_equatorial (is zenaz):   ", (t[0],t[1])
+        print "Should have been:                      ", (i[3], i[4])
+        print "Difference (dAZ, dALT) (arcsec):       ", ((t[0] - i[3]) / ARC_RAD, (t[1] - i[4]) / ARC_RAD)
+
+        print("\n")
+
+def oldvsnew_diagram():
```

Comment:
This 'test'/comparison/benchmark function should be in the scripts directory ( `/sapphire/scripts/...` )